apps/frontend-v3/app/(app)/build/page.tsx

```diff
@@ -0,0 +1,23 @@
+'use client'
+
+import { Stack, VStack } from '@chakra-ui/react'
+import { HeaderBanner } from '@repo/lib/modules/pool/actions/create/header/HeaderBanner'
+import { PoolForm } from '@repo/lib/modules/pool/actions/create/PoolForm'
+import { PoolPreview } from '@repo/lib/modules/pool/actions/create/PoolPreview'
+
+export default function BuildPage() {
```

Comment:
please add as much as possible into the component itself instead of here

Reply:
not sure what you mean? 

this pattern is same as lbp page and not intended to be final setup cus pon still deciding what build page will look like